snooty/types.py

```diff
@@ -466,7 +390,9 @@ def read(
         if match_found:
             for match in match_found:
                 lineno = text.count("\n", 0, match.start())
-                diagnostics.append(Diagnostic.error("git merge conflict found", lineno))
+                diagnostics.append(
+                    GitMergeConflictArtifactFound("git merge conflict found", lineno)
```

Comment:
This is always initialized with the same message, so let's factor this out as well